docs/monster-ui/spinner().md

```diff
@@ -0,0 +1,31 @@
+title: spinner()
+
+# monster.ui.spinner()
+
+## Syntax
+```javascript
+monster.ui.spinner(target[, options]);
+```
+
+### Parameters
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`$target` | Input element on which the method will be applied. | `jQuery` | | `true`
+`options` | List of options for [jQuery UI Spinner Widget][spinner]. | `Object` | | `false`
```

Comment:
You should notify which props are not overridable.

Reply:
Do you mean the props for `options`? If so, all of them are overridable, because any of the options for jQuery UI Spinner widget can be set.

Reply:
I believe `change` ([line](https://github.com/2600hz/monster-ui/pull/386/files#diff-602a66ca60a9e72a9cdcf92ae9ec90cdR3543)) isn't, no?

Reply:
The default implementation set in the helper is not overridable itself, but a `change` prop can still be passed, and it will be called at the end of the default `change` function.

Reply:
Ah, you might document that behavior then.

Reply:
Yes, I added a description for it in the markdown document at 719dcd7

I described what the control does inside the `change` function, though I did not mention the `change` prop or event directly. Is it OK?

Reply:
You could me more specific and add an extra `options.change` entry in the parameters' table with a description such as:

> Callback on value change (note that this function if specified will not override the widget's behavior but can be used to retrieve the new value).

Reply:
Done :+1: